source/base/Text.ooc

```diff
@@ -53,6 +53,30 @@ Text: cover {
 	operator != (other: String) -> Bool { !(this == other) }
 	operator != (other: This) -> Bool { !(this == other) }
 	operator + (other: This) -> This { This new(this _buffer + other _buffer) }
+	operator + (other: Int) -> This {
+		converted := other toString()
```

Comment:
... and maybe using extensions isn't how you should do things, but keep in mind that this what `math.ooc` does.

It's a different story for `Cell`, because it's a generic class, and extending them doesn't work very well due to bugs in rock. Extending basic covers like `Int` is the *only* way you can define new functions on them, though, and there's not necessarily any harm in doing in it several places, I don't think. For instance, you could put `toText` functions for all of the numeric types in a single file named `toTextExtensions.ooc` or something, add that file to `ooc-base.use`, and (here's the bad part) add `use ooc-base` anywhere you need to call `toText` on a numeric type.